packages/react-core/src/components/Tile/Tile.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Tile/tile';
+import { css } from '@patternfly/react-styles';
+
+export interface TileProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the banner */
+  children?: React.ReactNode;
+  /** Additional classes added to the banner */
+  className?: string;
+  /** Title of the tile */
+  title: string;
+  /** Icon in the tile title */
+  icon?: React.ReactNode;
+  /** Flag indicating if the tile is selected */
+  isSelected?: boolean;
+  /** Flag indicating if the tile is disabled */
+  isDisabled?: boolean;
+  /** Flag indicating if the tile icon is stacked */
```

Comment:
Just a nit, but this might say "tile header is stacked" since that's the layout variation - though it does stack the icon, so that works, too :)